.editorconfig

```diff
@@ -0,0 +1,16 @@
+# EditorConfig is awesome: https://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+[*]
+indent_style = space
+indent_size = 4
```

Comment:
actually ^^ I like indent_size=2

Reply:
I like even more consistency (and `setup.py` was 4 spaces indented :-P)

Reply:
But ok, let's try indent_size=2 for this repo..